File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domene/SykefraværsstatistikkNæringMedVarighet.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene;

import lombok.AllArgsConstructor;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene.Sykefraværsstatistikk;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class SykefraværsstatistikkNæringMedVarighet implements Sykefraværsstatistikk {
  private int årstall;
  private int kvartal;
  private String næringkode;
  private String varighet;
  private int antallPersoner;
  private BigDecimal tapteDagsverk;
  private BigDecimal muligeDagsverk;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/SykefraværsstatistikkNæringMedVarighet.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

import java.math.BigDecimal

data class SykefraværsstatistikkNæringMedVarighet(
    override val årstall: Int = 0,
    override val kvartal: Int = 0,
    val næringkode: String? = null,
    val varighet: String? = null,
    override val antallPersoner: Int = 0,
    override val tapteDagsverk: BigDecimal,
    override val muligeDagsverk: BigDecimal,
) : Sykefraværsstatistikk

